salt/modules/system.py

```diff
@@ -168,7 +168,10 @@ def has_settable_hwclock():
     salt '*' system.has_settable_hwclock
     '''
     if salt.utils.which_bin(['hwclock']) is not None:
-        res = __salt__['cmd.run_all'](['hwclock', '--test', '--systohc'], python_shell=False)
+        res = __salt__['cmd.run_all'](
```

Comment:
If the hardware clock is not present, does the if statement above just need to be better refined?

Reply:
The case that this is trying to handle is that even though the `hwclock` binary is present on the filesystem, the computer hardware itself doesn't contain a hardware clock. So I am not sure what you are suggesting.

Reply:
Oh, OK. Thanks for the clarification.